Sources/Danger/Danger.swift

```diff
@@ -63,8 +67,8 @@ final class DangerRunner {
 
 // MARK: - Public Functions
 
-public func Danger() -> DangerDSL {
-    return DangerRunner.shared.dsl
+@inlinable public func Danger() -> DangerDSL {
```

Comment:
<!--
  1 failure:  Function name sho...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: Sources/Danger/Danger.swift;
  Line: 70;
-->
- :no_entry_sign: Function name should start with a lowercase character: 'Danger()' (`identifier_name`)